src/main/java/org/mockito/ArgumentMatcher.java

```diff
@@ -125,4 +132,12 @@
      * @return true if this matcher accepts the given argument.
      */
     boolean matches(T argument);
+
+    /**
+     * Provide a meaningful description of the matcher in error messages.
+     *
+     * @return a description of the values that the matcher matches, e.g. the
+     * {@link ArgumentMatchers#anyString()} matcher returns {@code "<any string>"}.
+     */
+    String toString();
```

Comment:
No it is not a breaking change. Any class that implements it will extend Object so it will get an implementation of that method. I simply added it to the interface to document that it is needed.